tools/batt-testing.py

```diff
@@ -0,0 +1,160 @@
+from asyncio import base_tasks
+from json import load
+from lib2to3.pgen2.pgen import DFAState
+import sys
+from time import time
+import pyulog as ulog
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.optimize import curve_fit
+
+
```

Comment:
let's use argparse. We should have the following input arguments:
* `-o  <1C_file>.ulog` or `--one-c  <1C_file>.ulog`
* `-t  <30A_file>.ulog` or `--thirty-amp  <30A_file>.ulog`
* `-c <capacity_mah>` or `--capacity <capacity_mah>`

Reply:
reference: https://docs.python.org/3/library/argparse.html